Content.Server/Automod/AutomodManager.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Content.Server.Database;
+using Content.Shared.Automod;
+using Content.Shared.Database;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Automod;
+
+public sealed class AutomodManager : IAutomodManager, IPostInjectInit
+{
+    [Dependency] private readonly IPrototypeManager _protoMan = default!;
+    [Dependency] private readonly IEntityManager _entMan = default!;
```

Comment:
A core out-of-simulation system like this should not have a dependency on the entity manager.

Reply:
I need a bit of help with this. Do I add it as a dependency in the ITextAutomodAction implementations and inject dependencies on each? Or is there some other architecture I should be using?

Reply:
I mean honestly, given that the only thing you even *have* this entity manager for is to send popups to the cursor... which makes no sense (other review comment)... just nuke it?